test/unit/recipe/thirdpartypasswordless/signInUp.test.tsx

```diff
@@ -57,6 +57,7 @@ describe("ThirdPartyPasswordless.SignInAndUp", () => {
         });
 
```

Comment:
we need to add an e2e test that uses SessionAuth without calling the init function and make sure that it works (in the context of SSR)